docs/changelog.rst

```diff
@@ -7,6 +7,17 @@ Main branch
 Bug fixes and enhancements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Version 0.3.2.
+--------------
```

Comment:
You can put your modification in "Main branch" for now. I am not sure what version number we should use yet.

Reply:
sounds good. thanks.